main.go

```diff
@@ -90,14 +91,14 @@ func (c *collector) Collect(ch chan<- prometheus.Metric) {
 	metrics, err := c.queryOpenTSDB()
 	if err != nil {
 		log.Errorf("Error scraping target %s: %s", c.target, err)
-		ch <- prometheus.NewInvalidMetric(prometheus.NewDesc("api_error", "Error scraping target", nil, nil), err)
+		ch <- prometheus.NewInvalidMetric(prometheus.NewDesc("api_error", "", nil, nil), errors.New("Error scraping target, check exporter logs"))
```

Comment:
You already create an error message above in log. Why not just:
```
err2 = errors.New("Error scraping target %s: %s", c.target, err)
log.Error(err2)
ch <- prometheus.NewInvalidMetric(prometheus.NewDesc("api_error", "", nil, nil), err2)
```

unless there is some security critical information in `err2`